packages/xstate-wallet/src/store/tests/outcomes-equal.test.ts

```diff
@@ -0,0 +1,26 @@
+import {outcomesEqual} from '../state-utils';
+import {SimpleAllocation} from '../types';
+import {AddressZero, HashZero} from 'ethers/constants';
+import {bigNumberify} from 'ethers/utils';
+
+const simpleAllocation1 = {
+  type: 'SimpleAllocation',
+  assetHolderAddress: AddressZero,
+  allocationItems: [{destination: HashZero, amount: bigNumberify('0x2')}]
+} as SimpleAllocation;
```

Comment:
Can you use a type declaration rather than assertion?

Reply:
Not as far as I know, since `Destination` is tagged: 

https://github.com/statechannels/monorepo/blob/6d3df1a56aadce744c980e502d73acaa30544dc0/packages/xstate-wallet/src/store/types.ts#L33

I suppose the assertion could be made on a finer grain level, at least.

Reply:
You could also export a `DestinationZero` constant from a test data file which has type `Destination`.